bin/test-cleanup

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+conduit_namespace=$1
```

Comment:
confirming that `test-run` defaults to `conduit` namespace but `test-cleanup` intentionally does not?